docs/sources/alerting/alerting-rules/alerting-migration/migration-api.md

```diff
@@ -0,0 +1,89 @@
+---
+description: Use the Grafana Alerting API import tool to convert your datasource managed alert rules into Grafana managed alert rules
+labels:
+  products:
+    - cloud
+    - enterprise
+    - oss
+title: Import data source-managed alert rules with Grafana Mimirtool
+menuTitle: API alert rules import
+weight: 601
+refs:
+---
+
+# Import data source-managed alert rules with Grafana Mimirtool
+
+You can convert data source-managed alert rules to Grafana-managed alert rules with the Grafana import tool in the Grafana user interface, or you can convert them with the Grafana Mimirtool command-line tool. This guide tells you how to use Mimirtool to import your data source-managed alert rules.
+
+## Before you begin
+
+To import data source-managed alert rules with Grafana Mimirtool, you need to have the Grafana Mimirtool command-line tool installed.
+
+You need a service account the following [RBAC permissions](/docs/grafana/latest/administration/roles-and-permissions/access-control/):
+
+- Alerting: Rules Reader
+- Alerting: Rules Writer
+- Alerting: Set provisioning status
+- Datasources: Reader
+- Folders: Creator
+- Folders: Reader
+- Folders: Writer
+
+You also need to create a service account token with your service account. Refer to the [documentation for more information on service accounts and service account tokens](/docs/grafana/latest/administration/service-accounts/)
+
+## How it works
+
+When data source-managed alert rules are converted to Grafana-managed alert rules, the following are applied to the Grafana-managed alert rules:
+
+- All rules are given `rule_query_offset` offset value of 1m.
+- The `missing_series_evals_to_resolve` is set to 1 for the new rules.
+- The newly created rules are given unique UIDs.  
+  If you don't want the UID to be automatically generated, you can specify a specific UID with the `__grafana_alert_rule_uid__` label.
+
+## Import alert rules with Mimirtool
+
+To convert your alert rules, use the following command prompt substituting the your URL and your service account token as indicated, followed by your intended Mimirtool command. For further reference, see the [Mimir HTTP API documentation](/docs/mimir/latest/references/http-api/#ruler-rules:~:text=config/v1/rules-,Get%20rule%20groups%20by%20namespace,DELETE%20%3Cprometheus%2Dhttp%2Dprefix%3E/config/v1/rules/%7Bnamespace%7D,-Delete%20tenant%20configuration) for more information about the Rule API points and examples of Mimirtool commands.
```

Comment:
Should we also link the mimirtool commands to https://grafana.com/docs/mimir/latest/manage/tools/mimirtool/#rules?